packages/dart_frog_cli/lib/src/command_runner.dart

```diff
@@ -77,7 +104,10 @@ ${lightYellow.wrap('Changelog:')} $changelogLink
 Run ${lightCyan.wrap('$executableName update')} to update''',
           );
       }
-    } catch (_) {}
+    } catch (_) {
```

Comment:
Maybe add a verbose log of the error?

Reply:
Good catch! I've been meaning to do this and keep forgetting 🤦